Remove all ORA1 XModule code

sarina opened the pull request:
Python, CSS, JS.

Does not include removing xmodule testing infrastructure.

To merge into #10845

sarina commented:
Tests will be removed in a separate PR as github diff limits are 10k lines and this is already at 8k.

adampalay commented:
@waheedahmed and @symbolist , you all probably want to see this

sarina commented:
There is a strategy to how this is being done, this is not the next PR to be merged. Please see the hipchat room slash n burn for updates.

sarina commented:
OK, _now_ this is one of the next PRs to be merged.

The strategy of how this is being tackled is explained here: #10845 

So far 3 PRs have been merged, there are 2 currently opened ones, and I expect there to be another 2-3 after these.

sarina commented:
I need to get more of the Python unit tests passing before this can merge (on the main trunk there are only 2 remaining failing unit tests)

symbolist commented:
@adampalay Finally! :)

dianakhuang commented:
:+1: from me on the deletions.

adampalay commented:
@sarina , idk if these are the tests that you're looking to fix, but it looks like the roundtrip tests are failing because there are test XML courses with combindedopenended blocks

:+1:  on the deletions. So, after this PR, there will be one more to fix and remove remaining tests?

sarina commented:
Yeah I'm working on a PR to start doing test fixes currently.

stvstnfrd commented:
+1

sarina commented:
@adampalay @stvstnfrd @dianakhuang I've added another commit.

Please grep through the common/ directory and make sure I'm not missing anything obvious.

dianakhuang commented:
Sweet! :+1:

stvstnfrd commented:
@sarina +1 to latest commit

sarina commented:
@doctoryes could I invoke you here? In
e7612d993b7e0f07a833b9071ac1c563838ab8cf I removed all ORA1 units from the
manual test course, which in turn caused tests in
`test_mongo_call_count.py` to fail. In a fixup commit I've fixed the tests
but would love you to double check that commit. Happy to chat in person
about the goals of these PRs and what "review" means here (or check out the
_slash n burn_ hipchat room)

On Tue, Dec 8, 2015 at 6:00 PM, Steven Burch <notifications@github.com>
wrote:

> @sarina <https://github.com/sarina> +1 to latest commit
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10847#issuecomment-163048528>.
>

sarina commented:
This is ready for final review. I need review on the ORA components and an expert from Platform specifically to review the mongo call count changes.

The one JS and 4 Bokchoy failures are known, and not due to this branch; they will be addressed separately.

adampalay commented:
:+1:  on the ora components

dianakhuang commented:
:+1: on the last commit.

stvstnfrd commented:
+1 to 2e1c493ea5c0d97f104026fb8c50fe1cc38e4cd6

sarina commented:
Got John's :+1: on the mongo calls commit (99a90649babdd4d029441c7ec3f4be80c7c13c13) - merging this.

doctoryes commented:
:+1:  !
